### PR TITLE
Added field options to pilosa import

### LINKED ISSUE #1623: Add support for specifying field keys to pilosa import command
### Success criteria (What criteria will consider this ticket closeable?)
Following options are added to `pilosa import`:
* `--field-min`
* `--field-max`
* `--field-cache-type`
* `--field-cache-size`
* `--field-time-quantum`

### PR DESCRIPTION
**Pull request by yuce**
## Overview

Fixes #1623 

## Pull request checklist

- [ ] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [ ] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [ ] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [ ] I have resolved any merge conflicts.
- [ ] I have included tests that cover my changes.
- [ ] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

File: cmd/import.go [hunk 1]
```diff
@@ -51,8 +51,13 @@ omitted. If it is present then its format should be YYYY-MM-DDTHH:MM.
...
 	flags.StringVarP(&Importer.Field, "field", "f", "", "Field to import into.")
-	flags.BoolVar(&Importer.IndexKeys, "index-keys", false, "use keys=true when creating an index")
-	flags.BoolVar(&Importer.FieldKeys, "field-keys", false, "use keys=true when creating a field")
+	flags.BoolVar(&Importer.IndexOptions.Keys, "index-keys", false, "specify keys=true when creating an index")
+	flags.BoolVar(&Importer.FieldOptions.Keys, "field-keys", false, "specify keys=true when creating a field")
+	flags.Int64Var(&Importer.FieldOptions.Min, "field-min", 0, "specify the minimum for an int field on creation")
+	flags.Int64Var(&Importer.FieldOptions.Max, "field-max", 0, "specify the maximum for an int field on creation")
+	flags.StringVar(&Importer.FieldOptions.CacheType, "field-cache-type", "", "specify the cache type for a set field on creation")
+	flags.Uint32Var(&Importer.FieldOptions.CacheSize, "field-cache-size", 0, "specify the cache size for a set field on creation")
+	flags.Var(&Importer.FieldOptions.TimeQuantum, "field-time-quantum", "specify the time quantum for a time field on creation")
```
Comment (jaffee) [inline cmd/import.go]: it isn't obvious from the generated help that time quantum is a string, or what the format should be. let's at least add an example to this help string like `(e.g. "YMDH")`

File: cmd/import_test.go [hunk 2]
```diff
@@ -47,6 +49,20 @@ field = "f1"
 				return v.Error()
 			},
 		},
+		{
+			args: []string{"import", "--index", "i1", "--field", "f1", "--field-keys", "--field-max", "100"},
```
Comment (jaffee) [inline cmd/import_test.go]: I'd like to see tests that use all the new flags - especially time quantum since that one is a bit weird.

Comment (yuce) [reply cmd/import_test.go]: Updated

File: cmd/import.go (hunk 1 above)
Comment (yuce) [reply cmd/import.go]: Updated

**Comment by yuce** [comment]
@jaffee updated

**Comment by jaffee** [comment]
weird... I thought I needed to re-approve it, but I guess not

**Pull request merged**